sdk/attestation/azure-security-attestation/vcpkg/portfile.cmake

```diff
@@ -12,6 +12,7 @@ vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}/sdk/attestation/azure-security-attestation/"
     OPTIONS
         -DWARNINGS_AS_ERRORS=OFF
+        -DBUILD_TESTING=OFF
```

Comment:
Please add the same for `azure-core-amqp`, so that the PR fixes the issue across all packages, and the daily run gets fixed today.

Reply:
That's done in my current amqp PR.